src/Features/VisualBasic/Portable/FileHeaders/VisualBasicFileHeaderCodeFixProvider.vb

```diff
@@ -0,0 +1,29 @@
+﻿' Licensed to the .NET Foundation under one or more agreements.
+' The .NET Foundation licenses this file to you under the MIT license.
+' See the LICENSE file in the project root for more information.
+
+Imports System.Composition
+Imports Microsoft.CodeAnalysis.CodeFixes
+Imports Microsoft.CodeAnalysis.FileHeaders
+
+Namespace Microsoft.CodeAnalysis.VisualBasic.FileHeaders
+    <ExportCodeFixProvider(LanguageNames.VisualBasic, Name:=NameOf(VisualBasicFileHeaderCodeFixProvider))>
+    <[Shared]>
+    Friend Class VisualBasicFileHeaderCodeFixProvider
+        Inherits AbstractFileHeaderCodeFixProvider
+
+        Protected Overrides ReadOnly Property FileHeaderHelper As AbstractFileHeaderHelper
+            Get
+                Return VisualBasicFileHeaderHelper.Instance
+            End Get
+        End Property
```

Comment:
```suggestion
        Protected Overrides ReadOnly Property FileHeaderHelper As AbstractFileHeaderHelper =
            VisualBasicFileHeaderHelper.Instance
```

Reply:
Visual Basic doesn't support expression-bodied properties. This was intentionally a pass-through property as opposed to one with its own storage.

Reply:
so..  basically that is saving one pointers worth of data in hte entire app? :)   is that an important optimization?